tests/pexpect/boot.py

```diff
@@ -6,7 +6,7 @@
 
 
 def run() -> None:
-    p = pexpect.spawnu(" ".join(sys.argv[1:]), logfile=sys.stdout, timeout=240)
+    p = pexpect.spawnu(" ".join(sys.argv[1:]), logfile=sys.stdout, timeout=2048)
```

Comment:
Is this required? If so, let's add a comment explaining that this is for gentoo

Reply:
quick look into Action logs, related logs are lost but basically other distro would randomly fail in CI because of this. gentoo, even when it's using binary packages end up taking so much time that this timeout seems to be required. this is speculation however. i obviously don't have access to those jobs and how they get schedules on the CI servers.

*rebasing* after split-precursor is gonna take a while, i will run a test with lower timeout and put some pointers to logs, that is if they happen again.

Reply:
from [build log](https://github.com/257/mkosi/runs/3012721214?check_suite_focus=true#step:10:4):

``Failed to dissect image '/home/runner/work/mkosi/mkosi/mkosi.output/image.raw': Connection timed out``

Reply:
[(arch, gpt_btrfs)](https://github.com/257/mkosi/runs/3036790130?check_suite_focus=true#step:10:6)

Reply:
i kept it there so other distros don't timeout.
patch can't be merged without taking the binrepos.conf bit out:
```.github/workflow/ci.yml:113-117```

that's there to speedup the build (uses pre-compiled binaries) and it will go away once my ISP changes my ip. i have couple of solutions in mind for this but i rather see what's the direction you want to take. note that this doesn't need a running gentoo, just a plain http(s) or ssh server to hand out pre-compiled binary packages, for more see [here](https://wiki.gentoo.org/wiki/Binary_package_guide) [wiki.gentoo.org]